developer_manual/client_apis/OCS/ocs-taskprocessing-api.rst

```diff
@@ -0,0 +1,256 @@
++----------------------+--------+---------------------------------------------------------------------------------------------------------------+
+|``description``       | string | A description of the I/O slot                                                                                 |
++----------------------+--------+---------------------------------------------------------------------------------------------------------------+
+|``type``              | int    | The I/O slot type (See backend API for the available types)                                                   |
++----------------------+--------+---------------------------------------------------------------------------------------------------------------+
+|``mandatory``         | bool   | Whether this slot is mandatory or not                                                                         |
++----------------------+--------+---------------------------------------------------------------------------------------------------------------+
+
+Schedule a task
+---------------
+
+.. versionadded:: 30.0.0
+
+.. note:: The endpoint is rate limited as it can be quite resource intensive. Users can make 20 requests in 2 minutes, guests only 5
+
+* Method: ``POST``
+* Endpoint: ``/schedule``
+* Data:
+
++-----------------+-------------+--------------------------------------------------------------------------------+
+| field           | type        | Description                                                                    |
++-----------------+-------------+--------------------------------------------------------------------------------+
+|``input``        | array      | The input text for the task                                                     |
++-----------------+-------------+--------------------------------------------------------------------------------+
+|``type``         | string      | Id of this task's type.                                                        |
++-----------------+-------------+--------------------------------------------------------------------------------+
+|``appId``        | string      | The id of the requesting app                                                   |
++-----------------+-------------+--------------------------------------------------------------------------------+
+|``customId``   | string      | An custom app-defined identifier for the task (optional)                         |
++-----------------+-------------+--------------------------------------------------------------------------------+
+
+* Response:
+    - Status code:
+        + ``200 OK``
+        + ``400 Bad Request`` - When the task type is invalid
+        + ``412 Precondition Failed`` - When the task type is not available currently
+        + ``401 Unauthenticated`` - When the input references a file that the user doesn't have access to
+        + ``429 Too Many Requests`` - When the rate limiting was exceeded
+
+    - Data:
```

Comment:
Can it be:

```
Data (only in case of ``200 OK``):
```

without  multiple "Only provided in case of ``200 OK``" below

Reply:
there's a single entry for when it's not 200 OK, I copied this pattern from other OCS docs so :shrug: